docs/flink-connector.md

```diff
@@ -35,10 +35,8 @@ To create the table in Flink SQL by using SQL syntax `CREATE TABLE test (..) WIT
 
 * `connector`: Use the constant `iceberg`.
 * `catalog-name`: User-specified catalog name. It's required because the connector don't have any default value.
-* `catalog-type`: Default to use `hive` if don't specify any value. The optional values are:
-    * `hive`: The Hive metastore catalog.
-    * `hadoop`: The hadoop catalog.
-    * `custom`: The customized catalog, see [custom catalog](../custom-catalog) for more details.
+* `catalog-type`: `hive` or `hadoop` for built-in catalogs (defaults to `hive`), or left unset for custom catalog implementations using `catalog-impl`.
+* `catalog-impl`: The fully-qualified class name custom catalog implementation, must be set if `catalog-type` is unset. See also [custom catalog](../flink/flink-getting-started.md#custom-catalog) for more details.
```

Comment:
Nit: “The fully qualified class name _of a_ custom catalog implementation”. The addition of `of a` makes the sentence a bit more fluent.

Then I’d start a new sentence for “Must be set if `catalog-type` is unset”. But that part might just be my own opinion, but it reads a bit more clearly to me.

Reply:
makes perfect sense, I should have actually read the sentence when copy/pasting it from flink-connector.md :) I updated the sentence there as well